api/src/main/java/org/pmiops/workbench/access/AccessModuleServiceImpl.java

```diff
@@ -129,7 +128,7 @@ public List<AccessModuleStatus> getAccessModuleStatus(DbUser user) {
         .map(this::maybeReturnAccessModule)
         .filter(Optional::isPresent)
         .map(Optional::get)
-        .collect(Collectors.toList());
+        .toList();
```

Comment:
the changes in this file are just refactoring to use more recent Java features, as I happened to be looking at it